docker/metricbeat/statsd.yml

```diff
@@ -0,0 +1,9 @@
+# Module: statsd
+# Docs: https://www.elastic.co/guide/en/beats/metricbeat/main/metricbeat-module-statsd.html
+
+- module: statsd
+  # listen on all interfaces for testing purposes
+  host: "0.0.0.0"
```

Comment:
There is no actual risk by listening on all interfaces here since this is a test deployment. Production deployments could listen on the public interface only, of course